pandas/tests/test_nanops.py

```diff
@@ -1005,6 +1005,35 @@ def prng(self):
         return np.random.RandomState(1234)
 
 
+class TestNumpyNaNFunctions(object):
+
+    # xref GH 19629
+    methods_to_test = [
+        (np.sum, np.nansum),
+        (np.mean, np.nanmean),
+        (np.prod, np.nanprod),
+        (np.std, np.nanstd),
+        (np.var, np.nanvar),
+        (np.median, np.nanmedian),
+        (np.max, np.nanmax),
+        (np.min, np.nanmin),
+        (np.cumprod, np.nancumprod),
+        (np.cumsum, np.nancumsum)
+    ]
+
+    def test_np_nan_functions(self):
```

Comment:
parametrize these

Reply:
My bad, didn't realise you meant `pytest.mark.parametrize`, implemented your suggestion now 😄